### DISCUSSION TITLE
Simulation doesn't show water drops in sprinkler simulation

### DISCUSSION THREAD

**Discussion by gesamarie** [firemodels/fds#13448] [Q&A] [answered]
Hi everyone,
I have been trying to understand the key features of simulating sprinkler systems in FDS.
I have tested the implementation of my sprinklers in a small-scale simulation, where they work as they are supposed and water particles are simulated. If I copy the code fragment of the functioning sprinklers into my big simulation the simulation doesn't simulate any water particles. Does any argument in a FDS simulation prevent water particles from being simulated?

The sprinkler should be heat activated if it reaches a certain temperature (74°C)
For testing purposes, I set TMPA =80 in both simulations, so the sprinkler activates directly. 
One of my working sprinkler configurations looks like this:

&MISC TMPA=80.0, POROUS_FLOOR=.FALSE./
&SPEC ID='WATER VAPOR'/
&PART ID='Water',
      SPEC_ID='WATER VAPOR',
      DIAMETER=500.0,
      MONODISPERSE=.TRUE.,
      AGE=60.0,
      SAMPLING_FACTOR=1/
&PROP ID='Generic Industrial Spray01',
      PART_ID='Water',
      FLOW_RATE=100.0,
      PARTICLE_VELOCITY=5.0,
      SPRAY_ANGLE=15.0,85.0/
&DEVC ID='Sprinkler1', PROP_ID='Generic Industrial Spray01', XYZ=5.5,3.0,9.8, QUANTITY='TEMPERATURE', SETPOINT=74.0/

Any help would be highly appreciated, thank you in advance
Gesa

**Comment by drjfloyd** [comment]
We would need to see a case that doesn't work.

**Reply by gesamarie** [reply]
This code fragment works in my small test-simulation, but not in my bigger simulation. For me it doesn't really makes sense, that it doesn't work in the bigger simulation. The bigger simulation consists of more than one mesh. Could that cause that problem?
Or do you need to see the whole FDS File of the bigger Simulation?

**Reply by drjfloyd** [reply, marked as answer]
We can't diagnose an error in an input file that doesn't work by looking at an input file that does work. You likely have an error somewhere in the input file for your full simulation. One possible cause is your XYZ for your sprinkler is inside of a solid obstruction.

**Comment by Er9y714** [comment]
Are the mesh size the same in small and large scale simulations?
Do you see the sprinkler head activate in Smokeview (color change) in the large scale simulation? I think you can also monitor it in devc_ctrl csv and additional devices.
